src/utils/path.ts

```diff
@@ -1,12 +1,20 @@
 import { resolve, dirname, basename, relative, extname, join } from 'path'
 import { resolve as resolveUrl, parse as parseUrl, format as formatUrl } from 'url'
-import { TYPINGS_DIR, DTS_MAIN_FILE, DTS_BROWSER_FILE } from './config'
+import {
+  TYPINGS_DIR,
+  DTS_MAIN_FILE,
+  DTS_BROWSER_FILE,
+  MAIN_TYPINGS_DIR,
+  BROWSER_TYPINGS_DIR,
+  DEFINITIONS_DIR,
+  AMBIENT_DEFINITIONS_DIR
+} from './config'
 import isAbsolute = require('is-absolute')
 
-const mainTypingsDir = join(TYPINGS_DIR, 'main/definitions')
-const browserTypingsDir = join(TYPINGS_DIR, 'browser/definitions')
-const ambientMainTypingsDir = join(TYPINGS_DIR, 'main/ambient')
-const ambientBrowserTypingsDir = join(TYPINGS_DIR, 'browser/ambient')
+const mainTypingsDir = join(TYPINGS_DIR, MAIN_TYPINGS_DIR, DEFINITIONS_DIR)
```

Comment:
Moved declaration of folder names to the config. Looked at https://github.com/typings/core/blob/master/src/utils/path.ts#L166, but wasn't flexible enough for enumerating across all of the contents of the typings folder, at least not for the algorithm / solution that I implemented.